lock-api-objects/src/main/java/com/palantir/lock/LockResponse.java

```diff
@@ -164,6 +165,7 @@ public int hashCode() {
         return Objects.hash(token, lockHolders, isBlockAndRelease);
     }
 
+    @Unsafe
```

Comment:
I assume we only annotate here because the class itself is not serialisable, and thus the only way it can be unsafely represented is via this method?

Reply:
The return type of this is `String`, which isn't something we control